torch/distributions/binomial.py

```diff
@@ -75,7 +75,7 @@ def log_prob(self, value):
         log_factorial_k = torch.lgamma(value + 1)
         log_factorial_nmk = torch.lgamma(self.total_count - value + 1)
         return (log_factorial_n - log_factorial_k - log_factorial_nmk +
-                value * self.logits + (self.total_count - value) * torch.log1p(-probs))
+                value * self.logits + self.total_count * torch.log1p(-probs))

```

Comment:
Has someone reviewed this math? New version isn't equivalent to the old one

Reply:
I used to use `logits` here as `probs.log()`, now I fixed it to be log odds which is `probs.log() - torch.log1p(-probs))` , so they are equivalent. essentially, `value * old_logits - value * torch.log1p(-probs)) + self.total_count * torch.log1p(-probs))`  = `value * new_logits + self.total_count * torch.log1p(-probs))`

Reply:
@alicanb I trust tests more than manual inspection of code. Could you add some more test data in `test_binomial_log_prob()`, something like?
```py
probs = torch.arange(0.01, 1, 0.1) 
```
and check multiple total sizes `[0, 1, 2, 10]`?